Source/Editor/DualityEditor/DualityEditorApp.cs

```diff
@@ -234,7 +256,14 @@ public static void Init(MainForm mainForm, bool recover)
 			InitMainGraphicsContext();
 			DualityApp.InitPostWindow();
 
+			LoadEditorAppData();
+			SaveEditorAppData();
+			Directory.CreateDirectory(DualityApp.PluginDirectory);
```

Comment:
Directory.CreateDirectory already does a exist check internally so doesn't make sense to do this ourselves